tiledb/sm/array/array.h

```diff
@@ -731,6 +734,15 @@ class Array {
    * Note: the Sentry object will also be released upon Array destruction.
    **/
   void set_array_closed();
+
+  /** Obtain the Fragment's max tile for the named attribute or dimension */
```

Comment:
Add @param documentation.

Reply:
@params were added before -renamed- function which does not seem to be showing up in github app for me (below) here as expected... a bit confused about that, change showing in the branch for this pull request, don't know what's going on here.